waspc/ChangeLog.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## 0.19.0 (Unreleased)
```

Comment:
Btw why are we bumping to 0.19.0? Because of the new feature, `--db-image`?

@sodic @infomiho please correct me if I am wrong, but the scheme we have for versioning Wasp right now is that we treat that X in 0.X.Y as major version, and, and Y as minor version, and there is no patch version -> we also bump minor version then. So in this case, we actually woudn't bump it to 0.19, but 0.18.1. At least that is what I remember, but I am not 100% as I haven't been doing the PRs for some time, so asking @sodic and @infomiho to confirm this / comment on this.